src/hooks/useGetFollow.ts

```diff
@@ -0,0 +1,68 @@
+import { MouseEvent, useState } from 'react';
+
+import { createFollow, removeFollow } from '../apis/follow';
+import { getFollowingUser } from '../apis/getFollowingUser';
+import { userInfo } from '../apis/userInfo';
+
+interface List {
+  _id: string;
+  user: string;
+  fullName?: string;
+  image?: string;
+}
+
+const DUMMY_USER_ID = '63b9844b4a1b585b777da2ea';
+const useGetFollow = () => {
+  const [loading, setLoading] = useState(false);
+  const [followingIdList, setFollowingIdList] = useState<List[]>([]);
+
+  const getUserInfo = async () => {
+    try {
+      setLoading(true);
+      const newList: List[] = [];
+      await userInfo(DUMMY_USER_ID).then((res) => {
+        res.following.map(({ _id, user }: List) => {
+          newList.push({ _id: _id, user: user });
```

Comment:
```suggestion
          newList.push({ _id, user });
```
객체의 key랑 value의 이름이 같으면 value를 생략해도 된다고 합니당(단축 프로퍼티)

Reply:
기능구현에만 몰두해서 바꾸는걸 까먹었네요;;
리팩터링 할때 한번에 바꾸도록 하겠습니다!